### ISSUE TITLE
GMRT: Sound effect in platformer template continues to play even after backing out to the main menu

### ISSUE THREAD

**Issue by caitlinrmcintyre** [YoYoGames/GameMaker-Bugs#7679]
### Description

When running the platformer template in GMRT, I noticed after pressing play that a sound effect starts (alongside the music) but it isn't triggered by anything. This sound effect continues even after pressing ESC and backing out to the main menu.

 IDE v2024.1000.0.621  Runtime v2024.1000.0.647
 0.13.12

### Steps To Reproduce

1. Start GameMaker and create new project with the platformer template.
2. Run project in GMRT and press "Start".
3. Hear the sound effect play and press ESC and back out to main menu.
4. Notice the sound effect continues to play.

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GMRT are you reporting this issue for?

 0.13.12

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by toby-yoyo** [comment]
There are two parts to this one.

The sound heard alongside the music is the combination of the two ambient sounds emitted by the `obj_end_gate`.

The fact that it could be heard from the start of the level is an audio falloff bug that was fixed in an update to the `yyal` package.

However, with that fixed, if you navigate to the end gate so that the ambient sound can be heard and then back out to the main menu, you'll notice that the sound can still be heard. This is a separate bug caused by the clean up event of `obj_end_gate` not being called as it is on the GMS2 runtime, for which I've raised https://github.com/YoYoGames/GameMaker-Bugs/issues/7744.

Closing this issue as complete as the falloff bug has been fixed.
